kedro-airflow/features/steps/cli_steps.py

```diff
@@ -159,6 +159,8 @@ def create_project_from_config_file(context):
             str(context.config_file),
             "--starter",
             "pandas-iris",
+            "--checkout",
+            "0.18.0",
```

Comment:
This need to be revert before release

Reply:
Reverted, and this is actually not needed. `kedro-starter` would try to match kedro version automatically.